chore: update AppDynamics installation guide (#6687)
Kong/docs.konghq.com: app/_hub/kong-inc/app-dynamics/overview/_index.md

must be downloaded from the
[AppDynamics Download Portal](https://accounts.appdynamics.com/downloads).
The `libappdynamics.so` shared
library file is the only required file.

### Recommended installation

> If you are using Kong version 3.0.0.0 or later, the recommended way to installed the `libappdynamics.so` is placed it in the `/usr/local/kong/lib` directory.
> This directory is included in the Kong search path for shared libraries, so the `libappdynamics.so` file will be found automatically.

Reviewer: ```suggestion
If you are using {{site.base_gateway}} 3.0.0.0 or later, we recommended installing the `libappdynamics.so` in the `/usr/local/kong/lib` directory.
This directory is included in the {{site.base_gateway}} search path for shared libraries, so the `libappdynamics.so` file will be found automatically.
```

### Alternative installation

> If you are using an older version of Kong, or if you prefer to install the `libappdynamics.so` file in a different location, you can do so.

Reviewer: ```suggestion
If you are using an older version of {{site.base_gateway}}, or if you prefer to install the `libappdynamics.so` file in a different location, you can do so.
```

> - If Kong is deployed in RHEL or CentOS, the `libappdynamics.so` file can be placed in the `/usr/lib64` directory, which is included in the default search path for shared libraries.

Reviewer: ```suggestion
- If {{site.base_gateway}} is deployed in RHEL or CentOS, the `libappdynamics.so` file can be in the `/usr/lib64` directory, which is included in the default search path for shared libraries.
```

> - If Kong is deployed in Debian or Ubuntu, the `libappdynamics.so` file can be placed in the `/usr/lib` directory, which is included in the default search path for shared libraries.

Reviewer: ```suggestion
- If {{site.base_gateway}} is deployed in Debian or Ubuntu, the `libappdynamics.so` file can be in the `/usr/lib` directory, which is included in the default search path for shared libraries.
```
